internal/elasticsearch/securitycontextconstraints.go

```diff
@@ -55,7 +55,7 @@ func (er *ElasticsearchRequest) CreateOrUpdateSecurityContextConstraints() error
 
 	scc := builder.Build()
 
-	err := securitycontextconstraints.CreateOrUpdate(context.TODO(), er.ll, er.client, scc, securitycontextconstraints.Mutate)
+	err := securitycontextconstraints.CreateOrUpdate(context.TODO(), er.client, scc, securitycontextconstraints.Equal, securitycontextconstraints.Mutate)
```

Comment:
Why is the new parameter needed?

Reply:
Mainly "standardizing" it with the other request calls.

Reply:
Ok, now that you mention it. Wonder why we have it on the other methods... maybe for testing?

Reply:
Not entirely sure if this was just mimicking the controller-runtime implementation. However, we plan to move to that one, so having the two methods defined already will be helpful.